CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Unreleased
 
+## v4.0.0
```

Comment:
I doubt this would break any consumers, but I thought I would play it safe.

Reply:
reminder: when doing the releasing you have to create the tag directly on this `cisco/elsy`, and then don't touch the temp github release object. Travis will eventually overwrite it with the binaries when it finishes building the tag.